src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,124 @@
+            <FaFacebookF className="bg-gradient-to-r from-purple-500 to-pink-500 w-[30px] h-[30px] rounded-[50%] p-[7px]" />
+            <FaTwitter className="bg-gradient-to-r from-purple-500 to-pink-500 w-[30px] h-[30px] rounded-[50%] p-[7px]" />
+            <FaInstagram className="bg-gradient-to-r from-purple-500 to-pink-500 w-[30px] h-[30px] rounded-[50%] p-[7px]" />
+            <FaLinkedin className="bg-gradient-to-r from-purple-500 to-pink-500 w-[30px] h-[30px] rounded-[50%] p-[7px]" />
+          </div>
+        </div>
+        {data.map((item) => {
+          return (
+            <div key={item.id}>
+              <div className="mt-4 pl-6">
+                <h1 className="text-[#fff] font-bold text-[18px]">{item.h1}</h1>
+                <hr className="w-12 h-1 text-orange-300 mt-2 " />
+                <li className="mt-3 list-none grid  ">
+                  <a className="text-[#fff] list-none leading-8 " href="#">
+                    {item.a}
+                  </a>
+
+                  <a className="text-[#fff] list-none leading-8" href="#">
+                    {item.b}
+                  </a>
+
+                  <a className="text-[#fff] list-none leading-8 " href="#">
+                    {item.d}
+                  </a>
+
+                  <a className="text-[#fff] list-none leading-8 " href="#">
+                    {item.e}
+                  </a>
+
+                  <a className="text-[#fff] list-none leading-8 " href="#">
+                    {item.f}
+                  </a>
+
+                  <a className="text-[#fff] leading-8" href="#">
+                    {item.g}
+                  </a>
+                </li>
+              </div>
+            </div>
+          );
+        })}
+        <div className="mt-4">
+          <h1 className="text-[white]">Get In Touch</h1>
+          <hr className="w-12 h-1 text-orange-300 mt-2 " />
+          <div className="input flex border-white w-[350px] h-[55px] mt-5 ">
+            <input
+              className="item w-[340px] pl-2  h-[55px] mt-1 outline-none border-none  text-[white]"
+              type="text"
+              placeholder="Type Your Mail"
+            />
+            <button className="w-[100px] h-[40px] text-[white] p-1 pl-4 mt-3 pr-12 rounded-[5px] bg-gradient-to-r from-purple-500 to-pink-500 ">
+              Subscribe
+            </button>
+          </div>
+          <p className="text-[white] mt-8">
+            We Accepted <span className="text-[blue]">_____</span>
+          </p>
+          <div className="mt-6 flex gap-2">
+            <img
+              className=" w-[72px] h-[30px] cursor-pointer"
+              src="	https://hostim-html.themetags.com/assets/img/visa.png"
+              alt=""
+            />
+            <img
+              className=" w-[72px] h-[30px] cursor-pointer"
+              src="	https://hostim-html.themetags.com/assets/img/visa.png"
+              alt=""
+            />
```

Comment:
![image](https://user-images.githubusercontent.com/120715779/227906534-02821f1a-afc3-42ce-8e81-0335deb9d179.png)
mediani to'g'irla